CMakeLists.txt

```diff
@@ -201,6 +203,10 @@ if (MSVC)
     if(${flag_var} MATCHES "/INCREMENTAL" AND NOT ${flag_var} MATCHES "/INCREMENTAL:NO")
       string(REGEX REPLACE "/INCREMENTAL" "/INCREMENTAL:NO" ${flag_var} "${${flag_var}}")
     endif()
+    # Turn off warnings on Windows
+    string(REPLACE "/W3" "/w" ${flag_var} "${${flag_var}}")
```

Comment:
Would this help? It looks like the CI is using the release build.

Reply:
That would explain why this didn't work :)